src/noit_metric.h

```diff
@@ -101,6 +101,7 @@ typedef enum {
 } noit_message_type;
 
 #define NOIT_TAG_MAX_PAIR_LEN 256
+#define NOIT_NAME_MAX_PAIR_LEN MAX_METRIC_TAGGED_NAME + sizeof("__name:")
```

Comment:
- This is part of the public API and cannot change, i.e., you cannot change the value of this constant.  
- Beyond the API, and the ABI, this doesn't work either, because other than this one exception, we don't allow anything over 256 bytes anyway!  This constant must still reflect this otherwise we're allowing all tags to reach 4k

Reply:
this is a new definition. not a change to the old value

Reply:
Please use ``inline const`` for new stuff then